examples/shopping_ads/add_performance_max_retail_campaign.py

```diff
@@ -327,8 +305,8 @@ def create_campaign_criterion_operations(
     campaign_criterion.campaign = campaign_service.campaign_path(
         customer_id, _PERFORMANCE_MAX_CAMPAIGN_TEMPORARY_ID
     )
-    campaign_criterion.location.geo_target_constant = (
-        geo_target_constant_service.geo_target_constant_path("1023191")
+    campaign_criterion.location.geo_target_constant = geo_target_constant_service.geo_target_constant_path(
+        "1023191"
```

Comment:
Should we add a comment mentioning what the hardcoded IDs refer to, just as we do for  `# English`?